### PR TITLE
Add Fairwinds Insights to integrations

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -848,6 +848,34 @@ integrations:
...
+      - Docker
+      - Helm
+    tutorials:
+      - https://insights.docs.fairwinds.com/features/policy/
+      - https://insights.docs.fairwinds.com/reports/opa/
+      - https://insights.docs.fairwinds.com/features/admission-controller/
+      - https://insights.docs.fairwinds.com/features/continuous-integration/
+    videos:
+      - https://youtu.be/kmvPYjx1bpU
+      - https://youtu.be/kmvPYjx1bpU
```
Comment (ashutosh-narkar) [inline docs/website/data/integrations.yaml:871]: Nit: duplicate link

Comment (rbren) [reply docs/website/data/integrations.yaml]: Whoops, copy paste error. Fixed now. Thanks!

**Review by tsandall — APPROVED** [review]
~LGTM~ err actually, @rbren can you squash your changes? I'm not sure why there are 6 commits. Some of them are from clicking Update Branch but at least one is from your own updates. Once the commits are squashed me can merge!

**Comment by rbren** [comment]
Good call @tsandall - done.

**Review by anderseknert — APPROVED** [review]

**Pull request merged**